src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -40,7 +40,8 @@
     <UIFrameworkVersionMoniker Include="V4" />
   </ItemGroup>
 
-  <Target Name="BuildRazorViews" DependsOnTargets="Compile" BeforeTargets="Build">
+  <!-- Source build doesn't build this package -->
+  <Target Name="BuildRazorViews" DependsOnTargets="Compile" BeforeTargets="Build" Condition="'$(DotNetBuildFromSource)' != 'true'">
```

Comment:
Why is this necessary? This project isn't in Microsoft.AspNetCore.App

Reply:
So what happens is that when `ExcludeFromSourceBuild` is set is that noop'ing targets are imported and run during build: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Empty.targets. However, the way that these targets are defined, they still run explicitly when these projects are built, even though the Build target itself noops.